source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -38,39 +38,57 @@ There is currently support for the following device types within Home Assistant:
 - Select
 - Sensor
 
+{% include integrations/config_flow.md %}
+
+## Hub Capabilities
```

Comment:
```suggestion
## Hub capabilities
```
I realize heading style caps are used elsewhere on this page. But in general we use sentence-style capitalization also in headings.